wallet/models.go

```diff
@@ -579,27 +579,29 @@ type IncomingTransfersRequest struct {
 	// Return transfers for this account. (defaults to 0)
 	AccountIndex uint64 `json:"account_index,omitempty"`
 	// Return transfers sent to these subaddresses.
-	SubaddrIndices Index `json:"subaddr_indices,omitempty"`
-	// Enable verbose output, return key image if true.
-	Verbose bool `json:"verbose,omitempty"`
```

Comment:
The key image is always returned. At least in the current version of this API, there is no `verbose` option: https://github.com/monero-project/monero/blob/v0.18.2.2/src/wallet/wallet_rpc_server_commands_defs.h#L1146-L1148